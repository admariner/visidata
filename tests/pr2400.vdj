#!vd -p
{"sheet": "global", "col": null, "row": "disp_date_fmt", "longname": "set-option", "input": "%b %d, %Y", "keystrokes": "", "comment": null}
{"sheet": "global", "row": "disp_float_fmt", "longname": "set-option", "input": "%.05f", "keystrokes": ""}
{"sheet": "global", "row": "default_width", "longname": "set-option", "input": "50", "keystrokes": ""}
{"longname": "open-file", "input": "sample_data/test.fixed", "keystrokes": "o"}
{"longname": "open-file", "input": "sample_data/officials.jsonla", "keystrokes": "o"}
{"sheet": "test", "col": "", "row": "", "longname": "sheets-stack", "input": "", "keystrokes": "Shift+S", "comment": "open Sheets Stack: join or jump between the active sheets on the current stack"}
{"sheet": "sheets", "col": "", "row": "\u30adofficials", "longname": "open-row", "input": "", "keystrokes": "Enter", "comment": "open sheet referenced in current row"}
{"sheet": "officials", "col": "", "row": "", "longname": "sheets-stack", "input": "", "keystrokes": "Shift+S", "comment": "open Sheets Stack: join or jump between the active sheets on the current stack"}
{"sheet": "sheets", "col": "", "row": "\u30adsheets", "longname": "open-row", "input": "", "keystrokes": "Enter", "comment": "open sheet referenced in current row"}
{"sheet": "sheets", "col": "", "row": "\u30adofficials", "longname": "open-row", "input": "", "keystrokes": "Enter", "comment": "open sheet referenced in current row"}
{"sheet": "officials", "col": "Name", "row": "", "longname": "key-col", "input": "", "keystrokes": "!", "comment": "toggle current column as a key column"}
{"sheet": "officials", "col": "", "row": "", "longname": "sheets-stack", "input": "", "keystrokes": "Shift+S", "comment": "open Sheets Stack: join or jump between the active sheets on the current stack"}
{"sheet": "sheets", "col": "", "row": "\u30adtest", "longname": "open-row", "input": "", "keystrokes": "Enter", "comment": "open sheet referenced in current row"}
{"sheet": "test", "col": 0, "row": "", "longname": "key-col", "input": "", "keystrokes": "!", "comment": "toggle current column as a key column"}
{"sheet": "test", "col": "", "row": "", "longname": "sheets-stack", "input": "", "keystrokes": "Shift+S", "comment": "open Sheets Stack: join or jump between the active sheets on the current stack"}
{"sheet": "sheets", "col": "", "row": "\u30adtest", "longname": "stoggle-row", "input": "", "keystrokes": "t", "comment": "toggle selection of current row"}
{"sheet": "sheets", "col": "", "row": "\u30adofficials", "longname": "stoggle-row", "input": "", "keystrokes": "t", "comment": "toggle selection of current row"}
{"sheet": "sheets", "col": "", "row": "", "longname": "join-selected", "input": "merge", "keystrokes": "&", "comment": "merge selected sheets with visible columns from all, keeping rows according to jointype"}
