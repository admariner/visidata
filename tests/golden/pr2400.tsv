	a	b	c	Number	Exile
  	d  	 e 	  f		
1 	ggg	hhh	iiii		
 2	jjj	kkk	llllllll		
Daniel				1	True
Hananiah				2	True
Mishael				3	True
Azariah				4	True
Nebuchadnezzar				5	
